mlbench_core/models/pytorch/resnet.py

```diff
@@ -36,7 +36,7 @@ def batch_norm(num_features):
     """Create a batch normalization layer.
 
     See the Disclaimers in Kaiming's
-    `github repository <https://github.com/KaimingHe/deep-residual-networks/tree/a7026cb6d478e131b765b898c312e25f9f6dc031>`_
+    `here <https://github.com/KaimingHe/deep-residual-networks/tree/a7026cb6d478e131b765b898c312e25f9f6dc031>`_
```

Comment:
"See the disclaimer in Kaiming's here" <-- something more descriptive than "here" would be good :) or did you forget to also delete "In Kaiming's" ?